### ISSUE TITLE
Improvement of error messages in case of missing configuration files

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-testsuitesmanagement#82]
**Use case 1:**

Within a variant configuration file I refer to a not existing parameter configuration file:

`"name": "exercise-05_config_variant1.json.notexisting",`

I get a proper error message - together with:

```
Test Case exercise-05 :: exercise-05                                  | UNKNOWN |
Parent suite setup unknown:
Unknown exception occurs. Details: None
------------------------------------------------------------------------------
Exercise-05                                                           | UNKNOWN |
Suite setup unknown:
Unknown exception occurs. Details: None
```

**Use case 2:**

Within a parameter configuration file I import a not existing file:

`"[import]"   : "./exercise-05_config_common.json.notexisting",`

I also get a proper error message - together with this:

```
Test Case exercise-05 :: exercise-05                                  | UNKNOWN |
Parent suite setup unknown:
Unknown exception occurs. Details: json file 'C:/RobotTest/projects/ROBFW_AIO/robotframework-tutorial/900_building_testsuites/exercise-05/config/exercise-05_config_variant1.json': 'File 'C:\RobotTest\projects\ROBFW_AIO\robotframework-tutorial\900_building_testsuites\exercise-05\config\exercise-05_config_common.json.notexisting' is not existing!'
------------------------------------------------------------------------------
Exercise-05                                                           | UNKNOWN |
Suite setup unknown:
Unknown exception occurs. Details: json file 'C:/RobotTest/projects/ROBFW_AIO/robotframework-tutorial/900_building_testsuites/exercise-05/config/exercise-05_config_variant1.json': 'File 'C:\RobotTest\projects\ROBFW_AIO\robotframework-tutorial\900_building_testsuites\exercise-05\config\exercise-05_config_common.json.notexisting' is not existing!'
```

Summarized I get the full path of the not existing file 3 times. This is too much.

I would prefer to have:
* the content of the error messages aligned in both use cases
* the redundancy of the error messages reduced

**Comment by test-fullautomation** [comment]
postponed after 0.6.0 release

**Comment by namsonx** [comment]
Hello Holger,

For use case 2:

     Unknown exception occurs. Details: json file 'C:/RobotTest/projects/ROBFW_AIO/robotframework-tutorial/900_building_testsuites/exercise-05/config/exercise-05_config_variant1.json': 'File 'C:\RobotTest\projects\ROBFW_AIO\robotframework-tutorial\900_building_testsuites\exercise-05\config\exercise-05_config_common.json.notexisting' is not existing!'

The first json file is parent file which is contained the import file missing, second json file is the missing file. This helps users trace the issue more easier.

Thank you,
Son

**Comment by HolQue** [comment]
Below the full console output. It can be seen that **both** files are mentioned in error messages **three times**! This is rather much redundancy and it's confusing. It is enough to print an error message only once.

```
[ ERROR ] Loading of JSON configuration file failed! Reason: json file 'C:/RobotTest/projects/ROBFW_AIO/robotframework-tutorial/900_building_testsuites/exercise-06/config/exercise-06_config_default.json': 'File 'C:\RobotTest\projects\ROBFW_AIO\robotframework-tutorial\900_building_testsuites\exercise-06\config\exercise-06_config_common.json.notexisting' is not existing!'
Test Case exercise-06 :: exercise-06                                  | UNKNOWN |
Parent suite setup unknown:
Unknown exception occurs. Details: json file 'C:/RobotTest/projects/ROBFW_AIO/robotframework-tutorial/900_building_testsuites/exercise-06/config/exercise-06_config_default.json': 'File 'C:\RobotTest\projects\ROBFW_AIO\robotframework-tutorial\900_building_testsuites\exercise-06\config\exercise-06_config_common.json.notexisting' is not existing!'
------------------------------------------------------------------------------
Exercise-06                                                           | UNKNOWN |
Suite setup unknown:
Unknown exception occurs. Details: json file 'C:/RobotTest/projects/ROBFW_AIO/robotframework-tutorial/900_building_testsuites/exercise-06/config/exercise-06_config_default.json': 'File 'C:\RobotTest\projects\ROBFW_AIO\robotframework-tutorial\900_building_testsuites\exercise-06\config\exercise-06_config_common.json.notexisting' is not existing!'
```

**Comment by test-fullautomation** [comment]
Please refer to #96

**Comment by namsonx** [comment]
Hello Thomas,

The logging function was updated in the pull-request #116 

Thank you,
Son

**Comment by test-fullautomation** [comment]
solved and merged  with #116
